src/components/graphCard/graphCard.js

```diff
@@ -142,12 +142,19 @@ class GraphCard extends React.Component {
    * @returns {Node}
    */
   render() {
-    const { cardTitle, children, error, isDisabled, pending } = this.props;
+    const { cardTitle, children, error, graphData, isDisabled, pending, settings } = this.props;
 
     if (isDisabled) {
       return null;
     }
 
+    let hasActionDisplay = null;
+
+    // Apply hasActionDisplay callback, return boolean
```

Comment:
required: `hasActionDisplay` was a ref to an original bool, switch it out to just `actionDisplay`